helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
     app: {{ template "aerospike-kubernetes-operator.fullname" . }}
     chart: {{ .Chart.Name }}
     release: {{ .Release.Name }}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- range . }}
+  - name: {{ . }}
+  {{- end }}
+{{- end }}
```

Comment:
Can you add a commented out entry for `imagePullSecrets` in `values.yaml` with an example?

Reply:
Also document the configuration in README.

Reply:
Hello @spkesan I made the changes you requested in another commit; if it is ok for you, I will squash everything before the merge.